crates/cxx-qt-lib/src/gui/qvector4d.rs

```diff
@@ -147,6 +159,34 @@ impl std::fmt::Display for QVector4D {
     }
 }
 
+impl std::ops::Add for QVector4D {
+    type Output = Self;
+    fn add(self, other: Self) -> Self {
+        ffi::qvector4d_plus(&self, &other)
+    }
+}
+
+impl std::ops::Sub for QVector4D {
+    type Output = Self;
+    fn sub(self, other: Self) -> Self {
+        ffi::qvector4d_minus(&self, &other)
+    }
+}
+
+impl std::ops::Mul<f32> for QVector4D {
+    type Output = Self;
+    fn mul(self, rhs: f32) -> Self {
+        ffi::qvector4d_mul(rhs, &self)
+    }
+}
+
+impl std::ops::Div<f32> for QVector4D {
+    type Output = Self;
+    fn div(self, rhs: f32) -> Self {
+        ffi::qvector4d_div(rhs, &self)
+    }
+}
+
 impl From<crate::QVector3D> for QVector4D {
     /// Constructs a 4D vector from the specified 3D vector.
     /// The w coordinate is set to zero.
```

Comment:
my only other questions would then be

- [ ] Do we want Add for QStringList with QString ? However that is more a stream in C++ https://doc.qt.io/qt-6/qstringlist.html#operator-lt-lt there is a QStringList + QStringList though, but just wondered if `stringlist + string` would make any sense in Rust :thinking: 
- [ ] Do we want the Ops in QMargins(F) ?
- [ ] Do we want the Add/Div in QRect(F) with QMargins(F) ?

Reply:
Implemented the last two points. I don't think the first one would make much sense, as it is neither Rust-idiomatic (for example neither Rust's `Vec` nor `LinkedList` implement an operator like that) nor does the Qt API implement that plus operator.